### PR TITLE
OpenCADD-KLIFS: Add statement of need

### PR DESCRIPTION
**Pull request by dominiquesydow**
## Description
Add a statement of need for OpenCADD-KLIFS to our docs.
See preview here: https://opencadd.readthedocs.io/en/klifs-stat-of-need/

## Todos
  - [x] Copy "Statement of need" section from OpenCADD-KLIFS paper to separate RST file (`databases_klifs_statement_of_need.rst`)
  - [x] Add this file to `index.rst` as subsection of the OpenCADD-KLIFS section in our docs

## Questions
None.

## Status
- [x] Ready to go

### CODE REVIEW

**Review by ojeda-e — CHANGES_REQUESTED** [review]
Hi @dominiquesydow,
thanks for adding these changes. Just a minor point to address.
Would you please specify the target audience for your repo?

Please see my suggestion below.

File: docs/databases_klifs_statement_of_need.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
+Statement of need
+=================
+
+
+
+`OpenCADD-KLIFS <https://opencadd.readthedocs.io/en/latest/databases_klifs.html>`_ 
+(``opencadd.databases.klifs``) is a part of the `OpenCADD <https://opencadd.readthedocs.io/>`_ 
+package, a collection of Python modules for structural cheminformatics.
```
Comment (ojeda-e) [inline docs/databases_klifs_statement_of_need.rst:8]: Maybe adding a short sentence about the general target audience in the first paragraph? (suggested) 
That statement will help to identify potential users.

Comment (dominiquesydow) [reply docs/databases_klifs_statement_of_need.rst]: Good point, thanks! I added a short sentence about the general target audience at the very beginning and moved the more technical sentence shown above to the end of that paragraph.

Should I update the respective paper paragraph accordingly?

**Review by ojeda-e — APPROVED** [review]
Thanks for adding these changes @dominiquesydow.
Please add them to the paper as well. They are required to be in docs and paper, separately. 
Changes approved!

File: docs/databases_klifs_statement_of_need.rst (hunk 1 above)
Comment (ojeda-e) [reply docs/databases_klifs_statement_of_need.rst]: Yes, please!

**Review by ojeda-e — APPROVED** [review]

**Comment by dominiquesydow** [comment]
Hi @ojeda-e, thanks, I am updating the paper accordingly in #131.

**Pull request merged**